src/System.Management.Automation/engine/remoting/fanin/OutOfProcTransportManager.cs

```diff
@@ -650,7 +650,7 @@ internal override void Dispose(bool isDisposing)
                 {
```

Comment:
@iSazonov, your last commit had 2 failures in `PowerShell-CI-static-analysis`
Verify Markdown Links.Verify links in /home/vsts/work/1/s/CHANGELOG/6.2.md.https://portal.msrc.microsoft.com/en-US/security-guidance/advisory/CVE-2018-8256 should work
```powershell
retry of URL failed with error: Response status code does not indicate success: 502 (Bad Gateway).
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 117
117:                                     throw "retry of URL failed with error: $($_.Exception.Message)"

```
Verify Markdown Links.Verify links in /home/vsts/work/1/s/CHANGELOG/6.2.md.https://portal.msrc.microsoft.com/en-US/security-guidance/advisory/CVE-2018-8415 should work
```powershell
retry of URL failed with error: Response status code does not indicate success: 502 (Bad Gateway).
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 117
117:                                     throw "retry of URL failed with error: $($_.Exception.Message)"

```